syntax = "proto3";

message Request {
    string message = 1;
}

message Response {
    string message = 1;
}

service ExampleService {
    rpc SayHello (Request) returns (Response);
    rpc SayHelloClientStream (stream Request) returns (Response);
    rpc SayHelloServerStream (Request) returns (stream Response);
    rpc SayHelloByDicStream (stream Request) returns (stream Response);
}
